crates/re_viewer_context/src/space_view/space_view_class_impl.rs

```diff
@@ -89,17 +104,16 @@ impl<T: SpaceViewClassImpl> SpaceViewClass for T {
         ctx: &mut ViewerContext<'_>,
         ui: &mut egui::Ui,
         state: &mut dyn SpaceViewState,
-        scene: Scene,
+        scene: Box<dyn Scene>,
     ) {
-        let scene_elements = match T::SceneElementTuple::try_from(scene) {
-            Ok(scene_elements) => scene_elements,
-            Err(err) => {
-                re_log::error_once!("Incorrect scene type for space view class: {}", err);
+        let Some(typed_scene) = scene.as_any().downcast_ref()
```

Comment:
Don't worry about it for this PR, but I wonder if it could be worth dispatching to the `ui` calls through the `TypedScene` interface instead since in that context we have the type information to avoid these downcasts again.

Reply:
there's some more cruft coming on the ui call, and I'm not overly exited about it. Might need a pass which may also lead to that then 🤔